### PR TITLE
Fixes #9638: Update doc for the new server-node protocol

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9638

### CODE REVIEW

File: 23_manage_your_it/30_security/10_policy_security.txt [hunk 1]
```diff
@@ -30,15 +30,57 @@ More precisely:
...
+* The old protocol (CFEngine "1" protocol), which is plain text by default with the ability to encrypt certain file transfers,
+  kept for compatibility with older Rudder releases. It is deprecated and will disappear in future releases.
+* The new protocol (CFEngine "2" protocol), which is TLS based. Everything is encrypted, still using the CFEngine keys.
+
+Note: The new protocol was already available on the server since Rudder 2.11, but never used by the nodes.
+
+The access policy is:
+
+* Peer to peer key exchange, without central authority: keys are sent and accepted at first connection (from the policy server on nodes
+  and from nodes with an IP in the Allowed Networks on policy servers).
```
Comment (peckpeck) [inline 23_manage_your_it/30_security/10_policy_security.txt]: This is called TOFU (trust on first use)

File: 23_manage_your_it/30_security/10_policy_security.txt [hunk 2]
```diff
@@ -30,15 +30,57 @@ More precisely:
...
+* Node-specific files have an ACL containg the public key of the node, as found in the inventory
+* Common files have an IP-based ACL based on the Allowed Networks
+
+The old hostame and IP ACL are still generated for node-specific files to ensure compatibility with older nodes,
+but will be removed in the future.
+
+===== Inventory
+
+Nodes send in inventory to the server after installation or upgrade, and once a day.
```
Comment (peckpeck) [inline 23_manage_your_it/30_security/10_policy_security.txt]: "an inventory" would make it easier to read

File: 23_manage_your_it/30_security/10_policy_security.txt [hunk 3]
```diff
@@ -30,15 +30,57 @@ More precisely:
...
+
+This inventory contains various information, including:
+
+* The node's public key
+* The node's policy server
+
+The
```
Comment (peckpeck) [inline 23_manage_your_it/30_security/10_policy_security.txt]: The ?

File: 23_manage_your_it/30_security/10_policy_security.txt [hunk 4]
```diff
@@ -30,15 +30,57 @@ More precisely:
...
+Nodes send in inventory to the server after installation or upgrade, and once a day.
+
+This inventory contains various information, including:
+
+* The node's public key
+* The node's policy server
+
+The 
+
+* The inventories are sent by the node to its configured policy_server over HTTPS, currently whithout certificate validation.
```
Comment (peckpeck) [inline 23_manage_your_it/30_security/10_policy_security.txt]: Inventories

**Comment by amousset** [comment]
Commit modified

**Pull request merged**